airflow/migrations/versions/0132_2_8_0_add_priority_weight_strategy_to_task_.py

```diff
@@ -37,9 +37,18 @@
 
 
 def upgrade():
+    json_type = sa.JSON
+    conn = op.get_bind()
+    if conn.dialect.name != "postgresql":
+        # Mysql 5.7+/MariaDB 10.2.3 has JSON support. Rather than checking for
```

Comment:
I think we are not going to support Mysql 5 with Airflow 2.8? 
We already dropped support for it on main branch